script.js

```diff
@@ -0,0 +1,25 @@
+var boxes = document.getElementsByClassName("box");
+var boxNum = 2;
+
+for (var i=0; i<boxes.length; i++) {
+  boxes[i].addEventListener("click", checkMark);
+}
+
+function checkMark(event) {
+  if (boxNum % 2 === 0){
+    event.target.innerHTML = "X";
+  } else {
+    event.target.innerHTML = "O";
+  }
+  boxNum += 1;
```

Comment:
This is a clever way to keep track of who's turn it is. Can you think of a way to prevent people from overwriting previous turns taken?